src/components/TradeAssetSearch/components/FiatRow.tsx

```diff
@@ -1,20 +1,28 @@
 import { Box, Button, Flex, Text, useColorModeValue } from '@chakra-ui/react'
-import { useCallback } from 'react'
+import { useCallback, useMemo } from 'react'
 
 import { LazyLoadAvatar } from '@/components/LazyLoadAvatar'
+import type { FiatCurrencyItem } from '@/components/Modals/FiatRamps/config'
 import { getFiatFlagUrl } from '@/constants/fiatLogos'
-import type { FiatTypeEnumWithoutCryptos } from '@/constants/fiats'
 
 type FiatRowProps = {
-  fiat: FiatTypeEnumWithoutCryptos
-  onClick?: (fiat: FiatTypeEnumWithoutCryptos) => void
+  fiat: FiatCurrencyItem
+  onClick?: (fiat: FiatCurrencyItem) => void
 }
 
 export const FiatRow = ({ fiat, onClick }: FiatRowProps) => {
   const handleClick = useCallback(() => onClick?.(fiat), [fiat, onClick])
 
   const fiatNameColor = useColorModeValue('black', 'white')
 
+  const fiatName = useMemo(() => {
+    return fiat.name
+  }, [fiat.name])
+
+  const flagUrl = useMemo(() => {
+    return getFiatFlagUrl(fiat)
+  }, [fiat])
```

Comment:
can use shorthand if you feel fancy

Reply:
Changed in [622b3b2](https://github.com/shapeshift/web/pull/10592/commits/622b3b2a98efda4bd32ce27229e2d08fcceff629)